hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesResponse.java

```diff
@@ -35,10 +36,6 @@ default boolean isFailure() {
         return Optional.empty();
     }
 
-    default Optional<HermesMessage> getFailedMessage() {
```

Comment:
I'm not sure if it's a good idea to remove this method. It was added a few weeks ago and some users can already use this api. I think we can mark this method as deprecated if necessary, but we should keep it.

Reply:
ok i will leave it as deprecated method, now we always have response message not only in case of error

Reply:
We have no information what to use instead. What do you think about comment:
```
/**
 * @deprecated as of Hermes 1.2.4, in favor of {@link #getHermesMessage()}
 */
```
It would be helpful for user.